Update Evaluate.asm values

CounterPly opened the pull request:
All Evaluate.asm values are based on the following tuning patches:

https://github.com/official-stockfish/Stockfish/commit/917fe69f84ffcc53783e1ada41833684dc8a2970

CounterPly committed:
Update Evaluate.asm values

All Evaluate.asm values based on the following tuning patches:

https://github.com/official-stockfish/Stockfish/commit/917fe69f84ffcc53783e1ada41833684dc8a2970

tthsqe12 commented:
It is not good to apply the patches out of order. Out of order execution is too difficult for me to keep track of since I am concerned about identical functionality.

CounterPly committed:
Update Evaluate_Init.asm values

All modified values are based on the following tuning patches:

https://github.com/official-stockfish/Stockfish/commit/917fe69f84ffcc53783e1ada41833684dc8a2970

CounterPly commented:
Makes sense. I'll go back through the SF commits chronologically from now on prior to posting a pull request.
